src/Catty/ViewController/CatrobatTableViewController/CatrobatTableViewController.m

```diff
@@ -307,6 +314,34 @@ - (CGFloat)getHeightForCellAtIndexPath:(NSIndexPath*)indexPath
     return height; // for scrolling reasons
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
+    CGFloat height = size.height;
+    CGFloat width = size.width;
+    
+    if(self.statusBarHeight == 0) {
+        [coordinator animateAlongsideTransition:^(id<UIViewControllerTransitionCoordinatorContext> context) {}
+                                     completion:^(id<UIViewControllerTransitionCoordinatorContext> context) {
+            self.statusBarHeight = [self determineStatusBarHeight];
+            [TableUtil setStatusBar: self.statusBarHeight];
+            [self.tableView reloadData];
+        }];
+    }
+    
+    if (height >= width) {
+        [TableUtil setStatusBar: self.statusBarHeight];
+    } else {
+        [TableUtil setStatusBar:0];
+    }
+    [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
+}
+
+- (CGFloat)determineStatusBarHeight
```

Comment:
This is a very useful method. Could you please add it to `Util` directly so it can be reused? Also please rename it to `statusBarHeight`.